pytorch_lightning/metrics/classification/auroc.py

```diff
@@ -65,6 +65,17 @@ class AUROC(Metric):
            Callback that performs the allgather operation on the metric state. When ``None``, DDP
            will be used to perform the allgather
 
+    Raises:
+        ValueError:
+            If ``average`` is none of ``None``, ``"macro"`` or ``"weighted"``.
+        ValueError:
+            If max_fpr is not a ``float`` in the range ``0`` < ``max_fpr`` <= ``1``.
```

Comment:
@dipam7 Great addition! Could you follow @rohitgr7's suggestion in https://github.com/PyTorchLightning/pytorch-lightning/pull/6273#discussion_r585467010?
```suggestion
            If ``max_fpr`` is not a ``float`` in the range ``(0, 1]``.
```